fdbcli/RangeConfigCommand.actor.cpp

```diff
@@ -120,10 +120,30 @@ ACTOR Future<bool> rangeConfigCommandActor(Database cx, std::vector<StringRef> t
 	return true;
 }
 
+std::vector<const char*> rangeConfigGenerator(std::vector<StringRef> const& tokens, bool inArgument) {
+	if (tokens.size() == 1) {
+		return { "<show|set|update>", "[ARGS]" };
+	}
+	auto cmd = tokens[1];
+	if (cmd == "show"_sr) {
```

Comment:
Should we only print this once?

```suggestion
	if (cmd == "show"_sr && tokens.size() == 2) {
```

Reply:
added.